Cargo.toml

```diff
@@ -13,10 +13,12 @@ rust-version = "1.59"
 [dependencies]
 embedded-hal = { version = "0.2.6", features = ["unproven"] }
 nb = "1.0.0"
-atomic-polyfill = "1.0.2"
 riscv = { version = "0.10.1", features = ["critical-section-single-hart"] }
 e310x = { version = "0.11.0", features = ["rt", "critical-section"] }
 
+[target.'cfg(not(target_has_atomic = "32"))'.dependencies]
```

Comment:
It seems that this does not work with Rust 1.59... I'll keep bumping the MSRV until CI passes